tools/zos-update-worker/internal/update_worker.go

```diff
@@ -138,7 +138,7 @@ func (w *Worker) updateZosVersion(network Network, manager client.Manager) error
 	// zos light
 	if chainVersion.VersionLight != "" {
 		zosLightCurrent := fmt.Sprintf("%v/.tag-%v", w.src, chainVersion.VersionLight)
-		zosLightLatest := fmt.Sprintf("%v/%v-v4", w.dst, network)
+		zosLightLatest := fmt.Sprintf("%v/%v-v3light", w.dst, network)
```

Comment:
v4 won't exist? what about last versions?

Reply:
v4 now is for the new v4 and we still don't if it wil be handled in the same hub or will be a new hub

Reply:
the issue now is we need to reboot all zos4 nodes with version=v3light